README.md

```diff
@@ -1,9 +1,9 @@
 # Avanti Fellows OTP Service
```

Comment:
use the github workflows and docs as added here: https://github.com/avantifellows/plio-aws-discord-alerts